services/galaxytoolshed/galaxy-toolshed-downloads.service.js

```diff
@@ -0,0 +1,55 @@
+import Joi from 'joi'
+import { renderDownloadsBadge } from '../downloads.js'
+import BaseGalaxyToolshedService from './galaxy-toolshed-base.js'
+
+const repositoryRevisionInstallInfoSchema = Joi.array().items(
+  Joi.object({
+    times_downloaded: Joi.number().required(),
+  }).required(),
+  Joi.object({}),
+  Joi.object({})
+)
+
+export default class GalaxyToolshedDownloads extends BaseGalaxyToolshedService {
+  static category = 'downloads'
+  static route = {
+    base: 'galaxy-toolshed/downloads',
+    pattern: ':repository/:owner',
+  }
+
+  static examples = [
+    {
+      title: 'Galaxy Toolshed - Downloads',
+      namedParams: {
+        repository: 'sra_tools',
+        owner: 'iuc',
+      },
+      staticPreview: this.render({ downloads: 10000 }),
```

Comment:
Should call `renderDownloadsBadge` directly:

```suggestion
      staticPreview: renderDownloadsBadge({ downloads: 10000 }),
```

Reply:
Ok, it's modified 059ab9d18